tools/functions

```diff
@@ -15,7 +15,7 @@ MY_NAME="Firecracker $(basename "$ARG0")"
 #
 say() {
     [ -t 1 ] && [ -n "$TERM" ] \
-        && echo "$(tput setaf 2)[$MY_NAME]$(tput sgr0) $*" \
+        && echo "$(tput setaf 2)[$MY_NAME $(date -Is)]$(tput sgr0) $*" \
```

Comment:
nit: can we use -Iseconds?
on my machine I get an error "date: invalid argument 's' for -I" but "date -Iseconds" work on my machine and on the metal instance.

Reply:
Is that OSX version of the `date` command?